### ISSUE TITLE
Bengali transcription doesn't show bengali font

### ISSUE THREAD

**Issue by hafizsikder** [intel/openvino-plugins-ai-audacity#328]
![font](https://github.com/user-attachments/assets/b8ef03b2-6bbf-42db-86cf-4c8d0f0b22fe)
![settings](https://github.com/user-attachments/assets/254c2a1f-9d74-4778-a524-c39368f6565d)
![translation](https://github.com/user-attachments/assets/5806af34-8b32-4395-b9c8-14e396ffd633)
 
audio clip link: https://greenedubd-my.sharepoint.com/:u:/g/personal/hafiz_181001012_green_ac_bd/EWo4NJH_sKlNhjQDOaVOUjsBk2gR8mNGCbmYSSNZDDK9bQ?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=P50JhK 

This audio is 1 min and 30 sec long. 
The file I worked on was bigger. I updated both Audacity and this model. Same result. 
Thanks in advance for your kind support. 

I really need to transcribe these Bengali audio files.

**Comment by hafizsikder** [comment]
@RyanMetcalfeInt8, Hoping you are having a great day! In the audio, I recited the following lines, 
অনেকগুলো বছর পেরিয়ে যাবে,
তোমার সঙ্গে আমার আর দেখা হবে না।
এক শহরেই, অথচ দেখা হবে না।
পথ ভুলেও কেউ কারও পথের দিকে হাঁটবো না
আমাদের অসুখ বিসুখ হবে, দেখা হবে না।
কোনও রাস্তার মোড়ে, কিংবা পেট্রোল পাম্পে
কিংবা মাছের দোকানে, বইমেলায়, রেস্তোরাঁয়,
কোথাও দেখা হবে না।

আরও অনেকগুলো বছর পর, ভেবে রেখেছি
যেদিন হুড়মুড় করে একঝাঁক আলো নিয়ে
সন্ধ্যে ঢুকতে থাকবে আমার নির্জন ঘরে,
যেদিন বারান্দায় দাঁড়িয়ে আমার আঁচল উড়িয়ে
নিতে থাকবে বুনো বৈশাখী,
এক আকাশ চাঁদের সঙ্গে কথা বলবো যে রাতে সারারাত --
তোমাকে মনে মনে বলবোই সেদিন
কী এমন হয় দেখা না হলে?
দেখা না হলে মনে হতো বুঝি বেঁচে থাকা যায় না।
কে বলেছে যায় না, দেখ, দিব্যি যায়!

Translation: (the translation is not working either) 

"Many years will pass,
We'll never meet again, you and I.
In the same city, yet worlds apart,
Our paths will never cross.
We'll fall sick, but won't see each other.
At street corners, gas stations,
Fish markets, book fairs, restaurants,
Nowhere will we meet.

Years from now, I've imagined
The day evening rushes in,
A swarm of light, into my solitary room.
The day the wild monsoon wind
Will tug at my sari on the balcony,
On that night, talking to the moon all night,
I'll silently say to you,
What's so wrong with not meeting?
I thought I couldn't live without seeing you.
But who said you can't? See, I'm doing just fine!"

[trandlated using Gemini] 

fonts are in unicode, [You'll find Bengali fonts here: https://www.omicronlab.com/bangla-fonts.html] They are free to download and use. 

Kindly let me know if you need anything else.

**Comment by RyanMetcalfeInt8** [comment]
Hi @hafizsikder,

Sorry, I'm a little bit confused. By default, when I choose bengali as the source language, I get this:

![image](https://github.com/user-attachments/assets/81245d2e-1bc0-49f1-a553-adfbb4a3f7ce)

Is that what you also get? I'm trying to understand:

1. What you are getting by default (it looks like you change the font)?
2. What you want to get? 

Also, which whisper model are you using (base, small, medium, large)?

Thanks,
Ryan

**Comment by hafizsikder** [comment]
Hello @RyanMetcalfeInt8, I hope you have a great weekend. 

1. I got this in my output
![translation](https://github.com/user-attachments/assets/b8a422cb-37d7-4355-a121-987afe9c6a16)

2. What you got are some Bengali words in English Fonts, I need Bengali words in Bengali fonts. 

If you check my last reply, you'll get some idea of the topic and the fonts in Bengali Language. I added the English Meaning of the topic and a link to download the fonts if you want to explore. 

PS: I was extremely busy hence could not reply to you earlier! Kindly accept my sincere apology. 

here is my WhatsApp number if you want to connect directly! +8801915944557 

With the warmest regard, 
Md Hafiz Sikder

**Comment by hafizsikder** [comment]
Hey there! 
I have not changed any fonts prior to the output I shared with you. Audacity has "Nirmala" which is a Bengali font, I tried to change it after I got the output.

**Comment by RyanMetcalfeInt8** [comment]
So if you were to set output font to 'Nirmala', does it give the right output? Or is the *content* of the transcription also wrong?

**Comment by hafizsikder** [comment]
I am not sure I saw any option to change the font before running the model.

**Comment by RyanMetcalfeInt8** [comment]
> I am not sure I saw any option to change the font before running the model.

But you see option to change the font after the model runs, transcription is generated?

**Comment by hafizsikder** [comment]
Yes, from the bottom left panel, the 3 dots beside the word "Transcription", there is a potion to change the font. 
I left my PC at office, I could show you otherwise.

**Comment by RyanMetcalfeInt8** [comment]
> Yes, from the bottom left panel, the 3 dots beside the word "Transcription", there is a option to change the font.

Got it. So what I'm trying to understand is, once you change the font, is the result what you want? In other words, your request is to change the default font that is used?

**Comment by hafizsikder** [comment]
I understand, I completed my task manually anyway. I wanted you to know it because it could help millions like me. By the way, I tried larger models too. Same result. 
The model works on English audios just fine. 
Thank you anyway for putting so much effort on me. So kind of you. 
Have a beautiful day!
